scripts/flu_hosp_explore.R

```diff
@@ -104,54 +118,37 @@ data_targets <- make_data_targets()
 # These globals are needed by the function below (and they need to persist
 # during the actual targets run, since the commands are frozen as expressions).
 date_step <- 7L
-forecasts_and_scores_by_ahead <- make_forecasts_and_scores_by_ahead()
 forecasts_and_scores <- make_forecasts_and_scores()
-# ensembles
 
-# do a tar_map or tar_target to asdf
-ensembles_and_scores_by_ahead <- tar_map(
-  values = target_ensemble_grid,
-  names = id,
-  tar_target(
-    name = ensemble_by_ahead,
-    command = {
-      ensemble(joined_archive_data_2022,
-        forecaster_ids,
-        "hhs",
-        extra_sources = "chng",
-        ensemble_params,
-        ensemble_params_names
-      )
-    },
-    priority = .9999
-  ),
-  tar_target(
-    name = score_by_ahead,
-    command = {
-      run_evaluation_measure(
-        data = ensemble_by_ahead,
-        evaluation_data = hhs_evaluation_data,
-        measure = list(
-          wis = weighted_interval_score,
-          ae = absolute_error,
-          cov_80 = interval_coverage(0.8)
-        )
-      )
-    }
-  )
-)
-ensembles_and_scores <- make_ensemble_targets_and_scores()
-# other sources
+ensembles_and_scores <- make_ensembles_and_scores()
 external_names_and_scores <- make_external_names_and_scores()
 
 
 list(
```

Comment:
any particular reason one of these is a `list` and one is a `list2`?.

Reply:
i mostly end up using list2 whenever i get annoyed at list for not allowing trailing commas. they're interchangeable otherwise.